src/redux/categories/categories.js

```diff
@@ -0,0 +1,22 @@
+// Actions
+
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// Reducer
```

Comment:
- [ ] Great job defining the `CHECK_STATUS` as action type 👏🏻  but the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

- [ ] Same here, ensure you set an initial state with an empty array eg: `const initialState = [ ];`